src/components/Search/Search.stories.tsx

```diff
@@ -0,0 +1,21 @@
+// TODO: add search results
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Search from './Search';
+
+const meta = {
+  title: 'Components/Search',
+  component: Search,
+  parameters: {
+    layout: 'padded'
+  },
+  tags: ['autodocs']
+} satisfies Meta<typeof Search>;
+
+export default meta;
+type Story = StoryObj<typeof Search>;
+
+export const Playground: Story = {
+  render: (args) => <Search {...args} />
```

Comment:
Wasn't sure what to do in this case for text input too. since there aren't any args and render isn't neccesary I settled on `export const Playground: Story = {};` But that feels odd.